src/time.cr

```diff
@@ -19,7 +19,10 @@ require "crystal/system/time"
 # Time.new(2016, 2, 15) # => 2016-02-15 00:00:00
 #
 # # Specifying a time
-# Time.new(2016, 2, 15, 10, 20, 30) # => 2016-02-15 10:20:30 UTC
```

Comment:
This doc line was actually wrong, `Time.new(2016, 2, 15, 10, 20, 30)` currently returns a time of kind `Unspecified`, which does not have a `UTC` timezone.